core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3096,6 +3096,13 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).ok(expected);
   }
 
+  @Test public void testJsonLength() {
+    String query = "select json_length(\"product_name\", 'lax $') from \"product\"";
```

Comment:
I think it's better to add another case for the syntax `json_length(value)`.

Reply:
thanks,I have modified the test case.